assets/js/components/ClusterDetails/ClusterDetails.jsx

```diff
@@ -18,6 +23,12 @@ const siteDetailsConfig = {
       className: 'table-col-m',
       render: (content) => content?.join(', '),
     },
+    {
+      title: 'Virtual IP',
+      key: '',
+      className: 'table-col-m',
+      render: (_, _item) => 'Where to get this?',
```

Comment:
We need to add some logic to get this value, and most probably add the field in the HANA read model value object.
To get the value, we need to parse the list of cluster resources, find the one with `ocf::heartbeat:IPaddr2` type, and get the value. This resource is only available in one of the nodes.
Let's put a follow up task.

Reply:
Yup https://github.com/trento-project/web/issues/389